wallet/src/account/mod.rs

```diff
@@ -185,17 +272,19 @@ impl Account {
         amount: Amount,
         decomission_key: Option<PublicKey>,
     ) -> WalletResult<SignedTransaction> {
-        // process_send_request can fill UTXOs, but the first UTXO is needed in advance to calculate pool_id
-
-        // TODO: Add support for LockThenTransfer
-        let utxos = self
-            .get_utxos(UtxoType::Transfer.into())
-            .into_iter()
-            .map(|(outpoint, txo)| (outpoint, txo.clone()))
-            .collect::<Vec<(UtxoOutPoint, TxOutput)>>();
-
-        let input0 = utxos.get(0).ok_or(WalletError::NoUtxos)?;
-        let pool_id = pos_accounting::make_pool_id(&input0.0);
+        // the first UTXO is needed in advance to calculate pool_id, so just make a dummy Burn
+        // output and then replace it with the stake output
+        let request = SendRequest::new().with_outputs([TxOutput::Burn(OutputValue::Coin(amount))]);
```

Comment:
The `Burn` output takes up less space than the `CreateStakePool` output, so once it's replaced, the network fee calculation may be off. So I think it would be better to use a `CreateStakePool` output with a dummy `pool_id`.

Reply:
noticed also that replacing the outputs later with the proper pool_id will remove any change output that might have been added. For now, I changed it to find and modify the output, and plan to fix it properly in another PR when the fees are actually calculated properly. What do you think @pavel-kokolemin ?

Reply:
Great, thanks.